content/open-source/apko/getting-started-with-apko.md

```diff
@@ -114,64 +116,71 @@ Save and close the file after you're done including these contents. With `nano`,
-Jul 25 17:14:52.625 [INFO] [arch:x86_64] building image 'alpine-base:test'
-Jul 25 17:14:52.625 [INFO] [arch:x86_64] build context:
-Jul 25 17:14:52.625 [INFO] [arch:x86_64]   working directory: /tmp/apko-912607687
-Jul 25 17:14:52.625 [INFO] [arch:x86_64]   tarball path:
-Jul 25 17:14:52.625 [INFO] [arch:x86_64]   use proot: false
-Jul 25 17:14:52.625 [INFO] [arch:x86_64]   source date: 1970-01-01 00:00:00 +0000 UTC
-Jul 25 17:14:52.625 [INFO] [arch:x86_64]   Docker mediatypes: false
-Jul 25 17:14:52.625 [INFO] [arch:x86_64]   SBOM output path: /work
-Jul 25 17:14:52.625 [INFO] [arch:x86_64]   arch: x86_64
-Jul 25 17:14:52.625 [INFO] [arch:x86_64] image configuration:
-Jul 25 17:14:52.625 [INFO] [arch:x86_64]   contents:
-Jul 25 17:14:52.625 [INFO] [arch:x86_64]     repositories: [https://dl-cdn.alpinelinux.org/alpine/edge/main]
-Jul 25 17:14:52.625 [INFO] [arch:x86_64]     keyring:      []
-Jul 25 17:14:52.625 [INFO] [arch:x86_64]     packages:     [alpine-base]
-Jul 25 17:14:52.625 [INFO] [arch:x86_64]   cmd: /bin/sh -l
-Jul 25 17:14:52.625 [INFO] [arch:x86_64] doing pre-flight checks
-Jul 25 17:14:52.625 [INFO] [arch:x86_64] building image fileystem in /tmp/apko-912607687
-Jul 25 17:14:52.625 [INFO] [arch:x86_64] initializing apk database
-Jul 25 17:14:52.626 [INFO] [arch:x86_64] [cmd:apk] [use-proot:false] [use-qemu:] running: /sbin/apk add --initdb --arch x86_64 --root /tmp/apko-912607687
-Jul 25 17:14:52.732 [INFO] [arch:x86_64] initializing apk repositories
-Jul 25 17:14:52.732 [INFO] [arch:x86_64] initializing apk world
-Jul 25 17:14:52.732 [INFO] [arch:x86_64] initializing apk keyring
-Jul 25 17:14:52.739 [INFO] [arch:x86_64] synchronizing with desired apk world
-Jul 25 17:14:52.739 [INFO] [arch:x86_64] [cmd:apk] [use-proot:false] [use-qemu:] running: /sbin/apk fix --root /tmp/apko-912607687 --no-scripts --no-cache --update-cache --arch x86_64
-Jul 25 17:14:53.707 [INFO] [arch:x86_64] [cmd:/bin/busybox] [use-proot:false] [use-qemu:] running: /usr/sbin/chroot /tmp/apko-912607687 /bin/busybox --install -s
-Jul 25 17:14:53.737 [INFO] [arch:x86_64] generating supervision tree
-Jul 25 17:14:53.737 [INFO] [arch:x86_64] finished building filesystem in /tmp/apko-912607687
-Jul 25 17:14:53.837 [INFO] [arch:x86_64] built image layer tarball as /tmp/apko-temp-1954328382/apko-x86_64.tar.gz
-Jul 25 17:14:53.913 [INFO] [arch:x86_64] building OCI image from layer '/tmp/apko-temp-1954328382/apko-x86_64.tar.gz'
-Jul 25 17:14:53.981 [INFO] [arch:x86_64] OCI layer digest: sha256:d1354df643e7c9ad2e88ec505bf9566f8034038affafd7f119a4e269d1803d68
-Jul 25 17:14:53.981 [INFO] [arch:x86_64] OCI layer diffID: sha256:d34e58ab1ef2c3d4aff216116728afb18183fb4d6d4abe9a31ff83049a739524
-Jul 25 17:14:53.982 [WARNING] [arch:x86_64] multiple SBOM formats requested, uploading SBOM with media type: spdx+json
-Jul 25 17:14:53.984 [INFO] [arch:x86_64] output OCI image file to alpine-test.tar
+Feb  8 17:21:33.665 [INFO] loading config file: wolfi-base.yaml
```

Comment:
Maybe we can consolidate this output too, so that there is an ellipses at some point, something like:

```
Feb  8 17:21:33.666 [INFO] [arch:x86_64] WARNING: ignoring archs in config, only building for current arch (amd64)
Feb  8 17:21:33.666 [INFO] [arch:x86_64] building image 'wolfi-base:test'
...
Feb  8 17:21:42.064 [WARNING] [arch:x86_64] multiple SBOM formats requested, uploading SBOM with media type: spdx+json
Feb  8 17:21:42.068 [INFO] [arch:x86_64] output OCI image file to wolfi-test.tar
```